env/requirements.txt

```diff
@@ -23,3 +23,5 @@ matplotlib
 json2html
 requests
 opencv-python
+urllib3
```

Comment:
add these dependencies to the setup.py as well to make it available for pip deployments. (requirements.txt is for local development environment.)

Reply:
Done in: [5369aa5354c3187d741d69e6d9c2ebf2ba9de217](https://github.com/BxNxM/micrOS/pull/27/commits/5369aa5354c3187d741d69e6d9c2ebf2ba9de217#top)